inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -448,7 +446,6 @@ last_date <- dat %>%
 days_to_keep <- params$incomplete_days + params$r_estim_window
 i_recent <- dat_raw %>%
   incidence("date",
-    counts = count_var,
```

Comment:
So by this point, would this code work also with data that are pre-aggregated?

Reply:
No, I forgot to change this. Thanks!

Reply:
Fixed in [`9245e35` (#137)](https://github.com/epiverse-trace/episoap/pull/137/commits/9245e350048b29898b4b4c944dbf32b4ee012518)